src/authentication/types.ts

```diff
@@ -2,8 +2,7 @@ export const authServerBaseUrlInjectSymbol = Symbol('auth-server-base-url');
 export const gitlabRootPasswordInjectSymbol = Symbol('gitlab-root-password');
 export const jwtOptionsInjectSymbol = Symbol('token-verify-options');
 
-export const subEmailClaimKey: 'https://sub_email' = 'https://sub_email';
-export const teamTokenClaimKey: 'https://team_token' = 'https://team_token';
+export const teamTokenClaimKey: 'https://minard.io/team_token' = 'https://minard.io/team_token';
```

Comment:
Actually now that I change to the implicit form it behaves as you suspected. Weird, since I thought it didn't at some point.